client/gutenberg/editor/api-middleware/index.js

```diff
@@ -30,19 +30,28 @@ const debugMiddleware = ( options, next ) => {
 // Rewrite default API paths to match WP.com equivalents. Note that
 // passed apiNamespace will be prepended to the replaced path.
 export const wpcomPathMappingMiddleware = ( options, next, siteSlug ) => {
+	//support for fetchAllMiddleware, that uses url instead of path
```

Comment:
Curious 🤔 we may need to audit for anything else we're missing.

Reply:
I did introduce `url` mapping initially too, but removed it since it wasn't doing anything at the time.

What else might we be missing?